contracts/LifMarketMaker.sol

```diff
@@ -122,21 +176,21 @@ contract LifMarketMaker is Ownable {
     return block.number.sub(startBlock).div(blocksPerPeriod);
   }
 
-  function getSellRate() public constant returns (uint256) {
+  function getSellPrice() public constant returns (uint256) {
 
-    uint256 foundationWei = 0; //getFoundationWei();
+    uint256 periodIndex = getCurrentPeriodIndex();
 
-    uint256 currentMarketMakerWei = this.balance.sub(foundationWei);
+    uint256 sellPriceIncrement = initialSellPrice
+      .mul(PRICE_FACTOR.add(marketMakerPeriods[periodIndex].accumSellPriceIncrement))
+      .div(PRICE_FACTOR);
 
-    uint256 sellRate = lifToken.totalSupply()
-      .div(currentMarketMakerWei);
+    return sellPriceIncrement;
```

Comment:
The name of the variable is misleanding. `sellPriceIncrement` seems to be about some increment, but this is about the actual sellPrice. I'd just remove the var and return the value right away, that solves the naming issue :)